app/routes/rentals/show.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model(params) {
+    return this.store.queryRecord('rental', { title: params.title })
+  },
+
+  setupController(controller, model) {
```

Comment:
in other words, there's some valid use cases for `setupController` but they're fewer than most people think they are ;)